browser/src/map/handler/Map.Keyboard.js

```diff
@@ -430,6 +430,7 @@ L.Map.Keyboard = L.Handler.extend({
 
 					this._map.deselectAll();
 					this._map.setPart(partToSelect);
+					this._map._docLayer._preview._previewTiles[this._map._docLayer._selectedPart].focus();
```

Comment:
can we avoid using implementation details that deep in the structure?

Imagine we will do a refactor of slide previews, it will be less painful if we would use some high-level method outside. Like: `previews.focusCurrentPart()`

https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.PartsPreview.js